.ci/jenkins/project/Jenkinsfile.post-release

```diff
@@ -55,6 +55,19 @@ pipeline {
                 }
             }
         }
+        
+        stage('Prepare for post-release') {
+            steps {
+                script {
+                    dir(optaplannerRepository) {
+                        checkoutTag(optaplannerRepository, getProjectVersion(), stableBranchName)
+                    }
+                    dir(quickstartsRepository) {
+                        checkoutTag(quickstartsRepository, getGitTag(), stableBranchName)
```

Comment:
The stableBranchName is only for local checkout; it should take the release branch, right?

Reply:
```
void checkoutTag(String repo, String tagName, String localBranchName = tagName) {
    deleteDir()
    checkout(githubscm.resolveRepository(repo, getGitAuthor(), getBuildBranch(), false))
    // Need to manually checkout branch since we are in 'detached HEAD' state after the git checkout command.
    sh "git checkout tags/${tagName} -b ${localBranchName}"
}
```
This is just for the branch creation in the `checkoutTag`

I just moved the checkouts into a first stage. The logic remains the same